release-announcements/20210701-yt-4.0.0.txt

```diff
@@ -0,0 +1,81 @@
+The yt community is proud to announce the release of yt-4.0!
+
+yt (http://yt-project.org) is an open source, community-developed toolkit for analysis and visualization of volumetric data.
+
+If you are interested in contributing to yt, please see our repository on GitHub at http://github.com/yt-project/yt
+
+This release features a complete reworking of a number of internal components, particularly those related to particles and discrete points.
```

Comment:
To avoid repetition (particularly/particles)
```suggestion
This release features a complete reworking of a number of internal components,
most importantly those related to particles and discrete points.
```